spec/GraphQLOverHTTP.md

```diff
@@ -313,12 +324,18 @@ type (as indicated by the `Content-Type` header).
 If the client does not supply a `Content-Type` header with a POST request, the
 server SHOULD reject the request using the appropriate `4xx` status code.
 
+Note: This is left as a "should" to allow your server to choose a different
+behavior should it wish to.
```

Comment:
Is this needed?  As worded, it is no more than a casual definition of `SHOULD`.  If the intent is to reiterate and remind readers that a server can choose to vary behavior when a `SHOULD` is used, consider something like this:

```suggestion
Note: Rejecting such requests encourages clients to supply a `Content-Type` header with every POST request.  A server has the option to assume any media type they wish when none is supplied, with the understanding that parsing the request may fail.
```